src/gt4py/frontend/gtscript_frontend.py

```diff
@@ -361,28 +361,25 @@ def visit_FunctionDef(self, node: ast.FunctionDef):
         return node
 
 
-class ReturnReplacer(ast.NodeTransformer):
+class ReturnReplacer(gt_utils.meta.ASTTransformPass):
```

Comment:
Just a comment here completely unrelated to this PR: it would be worth to add a docstring to `gt_utils.meta.ASTPass` and `gt_utils.meta.ASTTransformPass`. It took me a while to remember that they are just clones of the standard `ast.NodeVisitor` and `ast.NodeTransformer` classes but forwarding the kwargs :facepalm: